hack/scripts/clean_private_cluster.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Have you checked if all these resources that are created will be noticed by our resource counter and sweeper?